src/rsa/signing.rs

```diff
@@ -219,6 +217,20 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         self.n_bits.as_usize_bytes_rounded_up()
     }
+
+    /// Exports the public modulus as big endian to the output array.
+    ///
+    /// The array should be of at least the size returned by
+    /// `public_modulus_len`.
+    pub fn export_public_modulus(&self, out: &mut [u8])
+                    -> Result<(), error::Unspecified> {
+        self.n.fill_be_bytes(out)
```

Comment:
`fill_be_bytes()` zero-pads the result. I think that JWT and similar things don't want zero-padded output, so there needs to be a check that `out` is exactly the right length. This check would make this symmetric with the `ring::signature::verify_rsa()` and other APIs exposed by *ring*.

Also, let's add unit tests that verify that we reject 1-byte-too-long and 1-byte-too-short `out`, and accept correct-length `out`.